ocgis/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+    name: ocgis
+    version: 1.2.0
+
+source:
+    git_url: https://github.com/NCPP/ocgis.git
+    git_tag: v1.2.0
+
+build:
+    number: 0
+    skip: True # [win]
+    skip: True # [osx]
+    skip: True # [py3k]
+
+requirements:
+    build:
+        - python >=2.7,<3
+        - cfunits-python
+        - fiona
+        - gdal <2
+        - libgdal <2
+        - netcdf4 <=1.1.6
```

Comment:
We can wait for a new release.  I was just wondering why.  Thanks for the explanation.

PS: This surfaced because I am re-building `gdal 1.11.3` and `fiona` and I noticed that, enforcing `netcdf4 <= 1.1.6` here, we end up with `numpy < 1.10`.  That happens because there is not netCDF4-python 1.1.6 compiled against a newer numpy.